integr8ly/integreatly-operator
# Mgdapi 3185

# Issue link
https://issues.redhat.com/browse/MGDAPI-3185 

# What
Added three new 3scale tests to verify the creation of Tenant, Backend and Product via CR.

# Verification steps
- Provision cluster
- Install RHOAM from https://github.com/integr8ly/integreatly-operator/pull/2409
- Run: 
``` 
INSTALLATION_TYPE=managed-api TEST=H08 make test/e2e/single BYPASS_STORAGE_TYPE_CHECK=true
```
```
INSTALLATION_TYPE=managed-api TEST=H09 make test/e2e/single BYPASS_STORAGE_TYPE_CHECK=true
```
```
INSTALLATION_TYPE=managed-api TEST=H10 make test/e2e/single BYPASS_STORAGE_TYPE_CHECK=true
```
- Will expect e2e on this pr to also pass.
